docs/blog/oras-0.14-and-future.md

```diff
@@ -0,0 +1,107 @@
+mkdir -p oras-install/
+tar -zxf oras_0.14.1_*.tar.gz -C oras-install/
+mv oras-install/oras /usr/local/bin/
+rm -rf oras_0.14.1_*.tar.gz oras-install/
+```
+
+> Note: You can also refer to the [installation guide](https://oras.land/cli/) for other different platforms.
+
+## Copy an image from Registry A to Registry B
+
+In this demo, assume all images are validated in MAR, so I will use ORAS to copy the container image from MAR to my personal repository of ACR. You can use your prefered container registry with ORAS.
+
+```
+oras copy mcr.microsoft.com/mmlspark/spark2.4:1.0.0 feynmanacr.azurecr.io/mmlspark/spark2.4:1.0.0
+```
+
+## Using SBOM Tool to generate a SBOM
+
+An SBOM creates a machine-readable inventory of the software components that make up a given software product. Generating SBOM is a first step in Supply Chain Security. 
+
+You can use [Docker SBOM](https://docs.docker.com/engine/sbom/) or [SBOM Tool](https://github.com/microsoft/sbom-tool) to generate a SBOM for the target image.
+
+SBOM Tool can be used to create SPDX 2.2 compatible SBOMs for any variety of artifacts. In this demo, we use SBOM Tool to create SPDX 2.2 compatible SBOM for the sample Spark image.
+
+Install SBOM Tool on a Linux machine:
+
+```
+curl -Lo sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
+chmod +x sbom-tool
+```
+
+Generate a SBOM for the Spark image stored in ACR:
+
+```
+sbom-tool generate -di feynmanacr.azurecr.io/mmlspark/spark2.4:1.0.0 -b ./foo -pn bar -pv 0.1 -bc ./foo -ps MyCompany -nsb http://mycompany.com
+```
+
+Then it will create a SBOM `manifest.spdx.json` in `foo/_manifest/spdx_2.2`.
+
+## Attach the SBOM to this image
+
+Next, let's attach the generate SBOM to this Spark image stored in ACR:
+
+```
+$ oras attach feynmanacr.azurecr.io/mmlspark/spark2.4:1.0.0 foo/_manifest/spdx_2.2/manifest.spdx.json --artifact-type example/sbom
+Uploading 97a5dc071dd1 manifest.spdx.json
+Uploaded  97a5dc071dd1 manifest.spdx.json
+Attached to feynmanacr.azurecr.io/mmlspark/spark2.4:1.0.0
+Digest: sha256:7592c8026675e463e7ced9b7ed369c2962b354a69b842423e8ctestdigest
+```
+
+## View the graph of artifacts
+
+A linked graph of supply chain artifacts can be viewed through the ORAS discovery command:
+
+```
+$ oras discover feynmanacr.azurecr.io/mmlspark/spark2.4:1.0.0
+Discovered 1 artifacts referencing feynmanacr.azurecr.io/mmlspark/spark2.4:1.0.0
+Digest: sha256:28de427f1df8cdb99bc98536b489d75cc496a2d37c3b9266248etestdigest
+
+Artifact Type   Digest
+example/sbom    sha256:7592c8026675e463e7ced9b7ed369c2962b354a69b842423e8ctestdigest
+```
+
+## ORAS Present and Future
+
+ORAS has been integrated and adopted by some industry-leading ISVs and projects, such as [soci-snapshotter](https://github.com/awslabs/soci-snapshotter) by AWS, [KubeApps](https://github.com/vmware-tanzu/kubeapps) by VMware Tanzu, [UOR Framework](https://universalreference.io/) by Red Hat etc. 
+
+TBD: Introduce ORAS 0.15 and future milestones.
```

Comment:
I will add this paragraph until https://github.com/oras-project/community/pull/21 is merged.